### DISCUSSION TITLE
모각코 기록을 관리하는 기능?

### DISCUSSION THREAD

**Discussion by ekgns33** [gdgoc-konkuk/platform-core#65] [Ideas]
디스코드 봇이랑 연동해서 모각코한사람들의 기록을 남겨보는게 어떨까요??

- 디스코드 봇을 만들게 될 것같은데 여러모로 활용가능성은 많아보임

**Comment by goldentrash** [comment]
오 좋은 제안입니다! Discord Web Hook을 사용한다는 아이디어죠?

**Reply by ekgns33** [reply]
이론상 그렇습니다 ㅋㅋㅋ
